lib/Doctrine/Common/Annotations/AnnotationReader.php

```diff
@@ -378,6 +410,32 @@ private function getPropertyImports(ReflectionProperty $property)
         return array_merge($classImports, $traitImports);
     }
 
+    /**
+     * Retrieves imports for constants.
+     *
+     * @param \ReflectionClassConstant $constant
+     *
+     * @return array
```

Comment:
Please be more descriptive about the elements of the array (use `Foo[]` if this is an array of foos)

Reply:
Returned annotations do not have a common ancestor. The best we can do is object[], but object is not valid in PHP 7.1. Therefore array is the best specification here.

Reply:
You could use `mixed[]`

Reply:
`mixed[]` means exactly the same as `array`.

Reply:
It conveys the fact that you did try to see if you could replace `array` with something better but could not.